src/plugins/home/public/application/components/home_list_card.tsx

```diff
@@ -64,39 +64,50 @@ interface Config {
 }
 
 export const HomeListCard = ({ config }: { config: Config }) => {
+  const isMobile = useIsWithinBreakpoints(['xs', 's', 'm', 'l']);
```

Comment:
Very nice to see responsive here, but I suppose we don't need a layout specifically for the mobile, just the desktop is enough. Because all workspace pages have not been adapted especially for mobile.

Reply:
thanks for the review, will update latter.